constants.js

```diff
@@ -2,3 +2,6 @@ export const MUX_HOME_PAGE_URL = 'https://mux.com?utm_source=stream-new';
 export const MUX_TERMS_URL = 'https://mux.com/terms?utm_source=stream-new';
 export const OPEN_SOURCE_URL = 'https://github.com/muxinc/stream.new';
 export const HOST_URL = 'https://stream.new';
+export const PLYR_TYPE = "plyr";
+export const MUX_PLAYER_TYPE = "mux-player";
```

Comment:
@davekiss I'm seeing a few places where `plyr` and `mux-player` is hard coded, maybe want to use these constants instead?